heidegger_index/views.py

```diff
@@ -22,6 +22,7 @@ class WorkDetailView(DetailView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
+        context["work_lemma"] = PageReference.objects.filter(lemma__value=context["work"].csl_json["title"], lemma__type="w")
```

Comment:
Zoals ik al zei, het zou mooi zijn als hij hier ook het `title-short` veld checkt, dan kunnen we via dat veld de naam van het lemma aan de reference koppelen, als bijvoorbeeld de titel erg lang is. Daarbij moet je even opletten dat `title-short` ook echt bestaat.

Je kunt dat bijv. doen door binnen `WorkDetailView` de volgende functie te schrijven:

```python
def _get_work_lemma(work: Work):
    short_title = work.csl_json.get("title-short")  # Geeft None terug als het veld niet bestaat
    if short_title:
        return PageReference.objects.filter(lemma__value=work.csl_json["title-short"], lemma__type="w")
    else:
        return PageReference.objects.filter(lemma__value=work.csl_json["title"], lemma__type="w") 
```

En die dan aan te roepen op deze regel:

```suggestion
        context["work_lemma"] = self._get_work_lemma(context["work"])
```

Reply:
Kan overigens ook gecomprimeerder, zonder extra functie, als volgt:

```suggestion
        context["work_lemma"] = PageReference.objects.filter(lemma__value=context["work"].csl_json.get("title-short") or context["work"].csl_json["title"], lemma__type="w")
```

Maar dat is een kwestie van smaak..